package.json

```diff
@@ -54,15 +54,15 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "warning": "^4.0.3",
-    "ilib": "^14.19.0"
+    "ilib": "^14.20.0"
   },
   "peerDependencies": {
     "ilib": "^14.19.0 || ^14.19.0-webos1"
```

Comment:
can you also update this peer dependency? https://www.npmjs.com/package/ilib-webos?activeTab=versions

Reply:
OK